controllers/preferencesController.js

```diff
@@ -0,0 +1,55 @@
+const jwt = require('jsonwebtoken');
+const Preference = require('../models/preference');
```

Comment:
Export Preference in `model/index.js`, then change this line to the following:
```suggestion
const { Preference } = require('../models');
```